### DISCUSSION TITLE
Add Terraform provider for easy migration between cloud stacks

### DISCUSSION THREAD

**Discussion by OK-Cactus** [ubicloud/ubicloud#1913] [Ideas]
Terraform provider support will make it easy to spin/migrate/import resources onto ubicloud

**Comment by fdr** [comment]
funny you mention it, there's an early version here, though, it's alphaware. It works, but we may change things, break things, etc.  https://github.com/ubicloud/terraform-provider-ubicloud/

you can try it out if you really want to, and maybe write to us at support@ubicloud.com and help us understand a bit better what you are trying to accomplish, so we can improve the software. It's totally optional, but we'd appreciate it.
